ImageLounge/src/DkCore/DkSettings.cpp

```diff
@@ -1078,8 +1078,9 @@ void DkSettingsManager::init()
 
     param().load(settings, true); // load defaults
 
-    int mode = settings.value("AppSettings/appMode", param().app().appMode).toInt();
-    param().app().currentAppMode = mode;
+    // TODO currentAppMode is obsolete, now set in main() after argument parsing
+    // int mode = settings.value("AppSettings/appMode", param().app().appMode).toInt();
+    // param().app().currentAppMode = mode;

```

Comment:
Should we remove this as currentAppMode is replaced?

Reply:
currentAppMode is still needed in a few places (should be removable with some refactoring), I'm commenting out since it could be useful to know when refactoring. At that time it should be removed.